RUFAS/biophysical/manure/single_stream_handler.py

```diff
@@ -0,0 +1,168 @@
+            self._om.add_error(
+                "None type ManureStream.",
+                "The processed ManureStream or pen data of the manure stream is None type.",
+                info_map,
+            )
+            raise TypeError("TypeError: Handler tries to process 'NoneType' object ManureStream.")
+        barn_temperature = self.determine_barn_temperature(conditions.mean_air_temperature)
+        surface_area = self.determine_barn_area(
+            self.manure_stream.pen_manure_data.animal_combination,
+            self.manure_stream.pen_manure_data.pen_type,
+            self.manure_stream.pen_manure_data.num_stalls,
+        )
+        self.ammonia_emission = self._calculate_ammonia_emissions(
+            self.manure_stream.ammoniacal_nitrogen,
+            self.manure_stream.volume,
+            990.0,
+            barn_temperature,
+            self.determine_ammonia_resistance(barn_temperature),
+            surface_area,
+            7.7,
+        )
+        return super().process_manure(conditions, time)
+
+    @classmethod
+    def determine_methane_emissions(
+        cls, animal_combination: AnimalCombination, pen_type: str, num_stalls: int, barn_temperature: float
+    ) -> float:
+        """
+        Calculates the methane housing emission.
+
+        Parameters
+        ----------
+        animal_combination : AnimalCombination
+            An AnimalCombination enum that describes the current animal makeup in this pen.
+        pen_type : str
+            The type of pen used for this pen.
+        num_stalls : int
+            Number of stalls.
+        barn_temperature : float
+            Temperature of the barn (Celsius).
+
+        Returns
+        -------
+        float
+            Methane emission from manure (kg).
+
+        """
+        barn_area = cls.determine_barn_area(animal_combination, pen_type, num_stalls)
+
+        return max(0.0, 0.13 * barn_temperature) * barn_area / 1000
+
+    @classmethod
+    def determine_carbon_dioxide_emissions(
+        cls, animal_combination: AnimalCombination, pen_type: str, num_stalls: int, barn_temperature: float
+    ) -> float:
+        """
+        Calculates the carbon dioxide housing emission.
+
+        Parameters
+        ----------
+        animal_combination : AnimalCombination
+            An AnimalCombination enum that describes the current animal makeup in this pen.
+        pen_type : str
+            The type of pen used for this pen.
+        num_stalls : int
+            Number of stalls.
+        barn_temperature : float
+            Temperature of the barn (Celsius).
+
+        Returns
+        -------
+        float
+            Carbon dioxide emission from manure (kg).
+
+        """
+        barn_area = cls.determine_barn_area(animal_combination, pen_type, num_stalls)
+        return max(0.0, 0.0065 + 0.0192 * barn_temperature) * barn_area / 1000
+
+    @staticmethod
+    def determine_ammonia_resistance(temp: float, hsc: float = 260) -> float:
```

Comment:
```suggestion
    def determine_housing_ammonia_resistance(temp: float, hsc: float = GasEmissionConstants.HOUSING_HSC) -> float:
```
If this is the wrong place to implement than HSC constant, please modify!

Reply:
Seems like the right place to me.